.mk/docker.mk

```diff
@@ -14,7 +14,7 @@
 
 BUILD_CONTAINERS=nsmd nsmdp nsmd-k8s vppagent-dataplane vppagent-dataplane-dev
 BUILD_CONTAINERS+=devenv crossconnect-monitor
-BUILD_CONTAINERS+=nsc icmp-responder-nse
+BUILD_CONTAINERS+=nsc nse
```

Comment:
Why the rename? Can you elaborate in the PR description?

Reply:
Also there are quite a few files changed with this rename, maybe it is a better idea to split these to another PR.

Reply:
Separate PR : #1013